tests/openvino/native/quantization/test_quantization_pipeline.py

```diff
@@ -120,3 +121,47 @@ def check_parameters(quantized_model, parameters, path):
     assert quantized_model.has_rt_info(base_path)
 
     check_parameters(quantized_model, quantize_parameters, base_path)
+
+
+@pytest.mark.parametrize(
+    "ignored_options, expected_dump",
+    [
+        (
+            IgnoredScope(names=["Conv_1", "Conv_2"]),
+            {"validate": None, "types": None, "subgraphs": None, "patterns": None, "names": "['Conv_1', 'Conv_2']"},
+        ),
+        (
+            IgnoredScope(names=["Transpose"], types=["Add"]),
+            {
+                "validate": None,
+                "types": "['Add']",
+                "subgraphs": None,
+                "patterns": None,
+                "names": "['Transpose']",
+            },
+        ),
+        (IgnoredScope(), {"": "[]"}),
+    ],
+)
+def test_ignored_scope_dump(ignored_options, expected_dump, tmp_path):
+    ignored_scpoe_path = ["nncf", "quantization", "ignored_scope"]
+
+    model = QuantizedModel().ov_model
+    dataset = get_dataset_for_test(model)
+    quantize_parameters = {
+        "preset": QuantizationPreset.PERFORMANCE,
+        "target_device": TargetDevice.CPU,
+        "subset_size": 1,
+        "fast_bias_correction": True,
+        "ignored_scope": ignored_options,
+    }
+    quantized_model = quantize_impl(model, dataset, **quantize_parameters)
+    ov.save_model(quantized_model, f"{tmp_path}_ov_model.xml")
```

Comment:
![image](https://github.com/openvinotoolkit/nncf/assets/4014476/942f748f-bec6-4256-aaea-2d6e30dfcc2e)
pytest prepared a separate folder for each test case, but the model is saved to a common directory. 
I'd do that:
```suggestion
    ov.save_model(quantized_model, tmp_path / "ov_model.xml")
```